ballerina/client.bal

```diff
@@ -58,13 +58,11 @@ public isolated client class Client {
     remote isolated function createTable(TableCreateInput tableCreationInput) returns TableDescription|error {
         string target = VERSION + DOT + "CreateTable";
         json payload = check tableCreationInput.cloneWithType(json);
-        convertJsonKeysToUpperCase(payload);
         map<string> signedRequestHeaders = check getSignedRequestHeaders(self.awsHost, self.accessKeyId,
                                                                         self.secretAccessKey, self.region,
                                                                         POST, self.uri, target, payload);
         map<json> response = check self.awsDynamoDb->post(self.uri, payload, signedRequestHeaders);
-        convertJsonKeysToCamelCase(response);
-        json tableDescription = check response.tableDescription;
+        json tableDescription = check response.TableDescription;
         return tableDescription.cloneWithType(TableDescription);
```

Comment:
Do we have to provide the type-definition to the `cloneWithType`, isn't it inferred ? And instead of cloneWithType isn't it meaningful to use `fromJsonWithType` here ?

Reply:
Fixed in [95b066a](https://github.com/ballerina-platform/module-ballerinax-aws.dynamodb/pull/49/commits/95b066abe7867ea2939f83aa2d4a41bdcfecc1cb)